newui/docs/_data/navigation.yml

```diff
@@ -0,0 +1,51 @@
+
+# main links
+main:
+  - title: "Documentation"
+    url: /docs/quick-start-guide.html
+  - title: "Community"
+    url: /community.html
+  - title: "Roadmap"
```

Comment:
lets keep the roadmap on the wiki .. and remove from the site..

Reply:
> High level, I think our goal here should be _strictly_ just match the current site as much as possible.. Land this first, have a link from current site to this new site.. left a few comments around this.
> 
> A week from now, switch to new site and link back to old site..
> A week from then, retire/remove the old site..
> Then we can make any content changes on top of the new site..

Hi @vinothchandar, I have a question that where I can place the new site.